assets/prototype/infrastructure/services/contextProviders/TicketProvider.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * External imports
+ */
+import { createContext, useState } from '@wordpress/element';
+
+export const TicketContext = createContext({});
+
+const TicketProvider = ({ children, id, relatedDates }) => {
```

Comment:
Same here.

Reply:
Removed that here: https://github.com/eventespresso/event-espresso-core/pull/1933/commits/c59201ef76b162f2b9f810dd6fcb334c424ef220